plugins/GSdx/GSSetting.cpp

```diff
@@ -181,6 +181,8 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Uploads GS data when rendering a new frame to reproduce some effects accurately. Fixes black screen issues in games like Armored Core: Last Raven.";
 		case IDC_MIPMAP_HW:
 			return	"Control the accuracy level of the mipmapping emulation\n\n"
+				"Automatic:\nAutomatically sets the recommended mipmapping level based on games that need mipmapping.\n"
```

Comment:
Maybe just "Automatically sets the mipmapping level based on the game." would do for this line? The repetition feels a bit awkward to me, but I dunno.

Reply:
I wanted for it to be a little more consistent with how the CRC Hack level tooltip is.

Reply:
Well, it's the "games that require mipmapping" part that I find awkward - I think it's somewhat implied already. IMO "recommended" is somewhat implied too - I don't think we're deliberately going to choose what is suboptimal.

Though perhaps it should be "based on whether the game is known to require mipmapping".